your-rents-core/pom.xml

```diff
@@ -81,6 +81,11 @@
             <artifactId>assertj-core</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
+            <version>2.21.0</version>
```

Comment:
shouldn't we define the version of the dependencies in the parent pom?

Reply:
Yes, you're right. Fixed by https://github.com/your-rents/your-rents-backend/pull/27/commits/80a54f2d14cd7a48e196787a9e6bd4ca09d5828b